### PR TITLE
Feature/93 google cloud llm

### PR DESCRIPTION
**Pull request by jtshark**
To see how to start the server, please look at ["Run Google Cl"](https://github.com/amosproj/amos2023ss03-qachat/wiki/Run-Google-Cloud-Platform/).

And please make sure to stop the server after you worked with it.

(Later we should add a cron job (or something like this) so that the LLM runs not all the time, but rather from 8am to 6pm or something like this @tkonheiser, @SaraElBrak )

### CODE REVIEW

**Comment by tkonheiser** [comment]
> To see how to start the server, please look at ["Run Google Cl"](https://github.com/amosproj/amos2023ss03-qachat/wiki/Run-Google-Cloud-Platform/).
> 
> And please make sure to stop the server after you worked with it.
> 
> (Later we should add a cron job (or something like this) so that the LLM runs not all the time, but rather from 8am to 6pm or something like this @tkonheiser, @SaraElBrak )

Issue has been created. [Issue 127](https://github.com/amosproj/amos2023ss03-qachat/issues/127)

File: QAChat/QA_Bot/deepL_translator.py
Comment (emuguy1) [inline QAChat/QA_Bot/deepL_translator.py:1]: Why did you move this? The deepL translation is also needed in the preprocessing? If we really split it up into multiple projects, we need to keep that in mind

**Comment by emuguy1** [comment]
Looks great. Please make sure to fix the conflicts with the main branch.

File: QAChat/QA_Bot/deepL_translator.py
Comment (jtshark) [reply QAChat/QA_Bot/deepL_translator.py:1]: yeeaaa what do you think of creating a "common" folder right to the "Data_Processing, QA_Bot and Slack_Bot" folder and putting there all code that is shared between parts of our project?

Comment (emuguy1) [reply QAChat/QA_Bot/deepL_translator.py:1]: Yes would be fine with that.
